operations/preinstall/Change_ServerTech_PDU_Credentials.md

```diff
@@ -0,0 +1,65 @@
+# Change Credentials on ServerTech PDUs
+
+This procedure changes password used by the `admn` user on ServerTech PDUs. 
+This procedure should be used to update all ServerTech PDUs in the system to 
+the same global credentials.
+
+**NOTE:** This procedure only updates the default credentials on the 
+ServerTech PDU hardware.  No credentials are set in any management software.
+
+## Prerequisites
+- The ServerTech PDUs must be accessible via a workstation or laptop.
+- Workstation or laptop has the 'curl' command installed.
+- The ServerTech PDU hostnames or IP addresses must be known.
+- The default 'admn' user password must be known for each PDU.
+
+## Procedure
+
+For each ServerTech PDU:
+
+1. Change password for the `admn` user on the ServerTech PDU. 
+    
+   ```bash
+   ncn-m001# curl -i -k -u admn:<OLD-PDU-PASSWORD> -X PATCH \
```

Comment:
I would change `ncn-m001#` to `linux#`, as this command be performed an any linux computer that has access to the BMC, and a ncn-m001 may not be still around from the last install.

Reply:
Done.